packages/react-generator/test/generator/expected/appdemo01/src/views/createVm/createVm.jsx

```
@@ -0,0 +1,481 @@
+              {state.dataDisk.map((item) => (
+                <div style={{ marginTop: '12px', display: 'flex' }}>
+                  <DatabaseOutlined style={{ marginRight: '10px', width: '16px', height: '16px' }}></DatabaseOutlined>
+                  <Select
+                    value={state.formData.diskType}
+                    onChange={(e) => setState((prev) => ({ ...prev, formData: { ...prev.formData, diskType: e } }))}
+                    placeholder="请选择"
+                    options={[
+                      { value: '1', label: '黄金糕' },
+                      { value: '2', label: '双皮奶' }
+                    ]}
+                    style={{ width: '200px', marginRight: '10px' }}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)